spyder/app/tests/test_mainwindow.py

```diff
@@ -3268,14 +3267,14 @@ def test_preferences_empty_shortcut_regression(main_window, qtbot):
     base_run_cell_advance = CONF.get_shortcut(
         'editor', 'run cell and advance')  # Should be Shift+Return
     base_run_selection = CONF.get_shortcut(
-        '_', 'run selection')  # Should be F9
+        'editor', 'run selection and advance')  # Should be F9
```

Comment:
Because this shortcut is defined under `editor` (and it has been like that from even before the changes done in this PR):

https://github.com/spyder-ide/spyder/blob/f10964010f66701597c40dba5e3203a54d9d9816/spyder/config/main.py#L488

I think I did this test change most probabIy since I have been running the tests locally and I stumbled across this one failing. Also, cheking this test definition more closely, I think is not being run on the CI. Most probably the reason is that the shortcut change (from `_` to `editor` + the shortcut string/name change) was done in the past but the test was not updated and instead skipped?. Should I also remove the CI skip for the test?